### PR TITLE
Fix Scrollbar Styling

### LINKED ISSUE #39: Scroll to bottom behaviour
### Current Behavior

When scrolling to the bottom of the page the layout is not aligned anymore

Screenshot 2024-08-21 at 10 48 55

### Reproduction / Steps To Reproduce

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mtwente**
# Pull request

## Proposed changes

* fix #39 
* add css to light mode scrollbar as well

## Types of changes

- [ ] New feature (non-breaking change which adds functionality).
- [x] Enhancement (non-breaking change which enhances functionality)
- [x] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have mentioned all co-authors in the PR description as `Co-authored-by: Name <email@domain.com>`.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `.sidebar-navigation` class to enhance styling and usability of sidebar navigation elements in both light and dark themes.
	- Improved scrollbar appearance with specified colors and scrolling behavior for better user experience.
	- Added a new `.footer` class to hide footer elements from the rendered view.

### CODE REVIEW

**Comment by maehr** [comment]
@mtwente The styling is nice, the scrollbar behavior is still the same due to the full width footer. Shall we turn of the footer?

**Comment by mtwente** [comment]
@maehr Ah, I didn't notice that. Then I'll just get rid of the footer.

File: styles.css [hunk 1]
```diff
@@ -67,6 +67,10 @@
 
 /*-- scss:defaults --*/
 
+.footer {
```
Comment (maehr) [inline styles.css:70]: @mtwente Can we turn off the footer via settings, see https://quarto.org/docs/websites/website-navigation.html#github-links

```
website:
  title: DigiHistCH24 Book of Abstracts
  favicon: logo.png
  open-graph: true
  twitter-card: true
  site-url: https://digihistch24.github.io/book-of-abstracts/
  repo-url: https://github.com/digihistch24/book-of-abstracts/
  # issue-url: https://github.com/digihistch24/book-of-abstracts/issues/new/choose
  # repo-actions: [edit, issue]
```

**Review by maehr — CHANGES_REQUESTED** [review]

File: styles.css (hunk 1 above)
Comment (maehr) [reply styles.css:70]: @coderabbitai is there a way to disable the footer on Quarto homepages?

Comment (maehr) [reply styles.css:70]: @mtwente We can always just hide it via css if there's no better solution. (Hiding elements still affects the DOM and therefore can cause performance issues)

**Review by maehr — APPROVED** [review]

**Pull request merged**